src/main/cliapp/src/containers/diseaseAnnotationsPage/NewAnnotationForm.js

```diff
@@ -176,8 +178,13 @@ export const NewAnnotationForm = ({
 							<FormErrorMessageComponent errorMessages={errorMessages} errorField={"negated"}/>
 						</SplitterPanel>
 						<SplitterPanel style={{paddingRight: '10px'}}>
+<<<<<<< Updated upstream
 							<label htmlFor="object"><font color={'red'}>*</font>Disease</label>
 							<AutocompleteEditor
+=======
```

Comment:
@kthorat-prog not sure how these got in there?

Reply:
Changes done as per comments above.